### ISSUE TITLE
UI Updates

### ISSUE THREAD

**Issue by islanddog** [MediaBrowser/Emby.Releases#62]
The Emby Tizen app doesn't update the UI until I leave and return to the page. This causes issues with LiveTV and when watching series as when I go back to a menu it shows unwatched or the wrong time. Just curious if this will be fixed in the upcoming update?

**Comment by LukePulverenti** [comment]
Hi, what version of the app are you on?

**Comment by islanddog** [comment]
Emby for Samsung 1.0.81

**Comment by LukePulverenti** [comment]
Ok. Can you give me an exact scenario including the set of steps taken in the user interface from the time you started up the app until the time the problem was experienced? Thanks.

**Comment by islanddog** [comment]
Definitely. When using it tonight I will keep an eye on it and take some screenshots. Essentially if I am watching a TV Show (not LiveTV) and I back out after a certain amount of time, sometimes I will be presented with the original watch screen and not asked to resume. If I back and go back into the show the Resume button will be visible.

Same thing on LiveTV. If I watch a TV program and back out to the guide sometimes I am presented with the guide when I originally put on the show. Even the clock in the top right shows the time originally shown and does not update until I click back and enter back into the LiveTV prompt.

**Comment by islanddog** [comment]
Another example last night (hard to show from the pictures I took). I watched Episode 5-8 of a show. When I selected back I was back on Episode 5 and it said play (not showing watched). When I returned to Home and back into the Show I was at 8 and Episode 5 showed as played.
